packages/specs/schema/src/components/open-spec/enumsMapper.ts

```
@@ -0,0 +1,20 @@
+import {JsonSchema} from "../../domain/JsonSchema.js";
+import {SpecTypes} from "../../domain/SpecTypes.js";
+import {JsonSchemaOptions} from "../../interfaces/JsonSchemaOptions.js";
+import {registerJsonSchemaMapper} from "../../registries/JsonSchemaMapperContainer.js";
+import {enumsMapper} from "../default/enumsMapper.js";
+
+export function wrapEnumsMapper(obj: any, schema: JsonSchema, options: JsonSchemaOptions) {
+  obj = enumsMapper(obj, schema, options);
+
+  if (obj.const) {
+    obj.enum = [obj.const];
+    delete obj.const;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)